src/lucky_web/status.cr

```diff
@@ -0,0 +1,63 @@
+module LuckyWeb
+  enum Status
```

Comment:
Hmm for some reason it did not save my comment. Here's roughly what I said:

I agree that `Status` is probably a bit ambiguous in this context, but if it was moved to `LuckyWeb::Action::Status` then I think it would make more sense, and you'd be able to use it like this in actions: `redirect to: "/foo", Status::MovedPermanently`